### PR TITLE
Initial import of a preliminary README.md for the *OpenCensus Agent f…

### PR DESCRIPTION
**Pull request by ubschmidt2**
…or Java*, grabbing the name and directory.

### CODE REVIEW

File: agent/README.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+(MongoDB vs. Redis, etc.) and the application.
+
+
+## Features
+
+The *OpenCensus Agent for Java* is in an early development stage. The following features will be
+implemented in the initial version:
+
+
+### Propagation of the trace context between threads
```
Comment (bogdandrutu) [inline agent/README.md]: s/trace//

File: agent/README.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+# OpenCensus Agent for Java
+
+[![Build Status][travis-image]][travis-url] [![Maven Central][maven-image]][maven-url]
+
+The *OpenCensus Agent for Java* collects and sends latency data about your Java process to
+OpenCensus backends such as Stackdriver Trace for analysis and visualization.
```
Comment (bogdandrutu) [inline agent/README.md]: I would say: Zipkin, Stackdriver, etc. for ...

File: agent/README.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+- java.sql.Statement#execute*
+
+The user-supplied configuration may request the instrumentation of additional, application-specific
+methods, e.g.:
+
+- io.opencensus.agent.example.*#main
+- io.opencensus.agent.example.*#doStuff
+
+
+### Propagation of the trace context between processes
```
Comment (bogdandrutu) [inline agent/README.md]: We would also like to propagate Tags. Probably better to say "Propagation of the context between processes"

Comment (ubschmidt2) [inline agent/README.md]: Done.

File: agent/README.md (hunk 1 above)
Comment (ubschmidt2) [inline agent/README.md]: Done.

File: agent/README.md (hunk 2 above)
Comment (ubschmidt2) [inline agent/README.md]: Done.

**Comment by adriancole** [comment]
Looks very nice. Some comments

* dont forward reference propagation. Rather update readme along with impl

* move design below features (even if features are todo). Very interesting
but knowing whats inside is probably higher priority

* there are notes at the bottom about loading strategy. If that isnt
implemented yet, similarly defer it to impl. Things documented with impl
are easier to review for parity with eachother

* one more time.. nice job! I especially like the intro

**Comment by ubschmidt2** [comment]
Thanks for the review comments. Moved to contrib/agent/. Also minor changes to the wording.

**Comment by ubschmidt2** [comment]
Thanks, Adrian, SGTM. I've removed the forward references and moved features above design.

File: contrib/agent/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+# OpenCensus Agent for Java
+
+[![Build Status][travis-image]][travis-url] [![Maven Central][maven-image]][maven-url]
```
Comment (bogdandrutu) [inline contrib/agent/README.md]: Copy from the main README?

Comment (ubschmidt2) [inline contrib/agent/README.md]: Done.

**Pull request merged**